tests/vcs/test_clone.py

```diff
@@ -217,3 +217,27 @@ def test_clone_unknown_subprocess_error(mocker, clone_dir):
             clone_to_dir=clone_dir,
             no_input=True
         )
+
+
+def test_git_clone_does_not_change_line_endings_on_win():
+    repo_dir = vcs.clone(
```

Comment:
This is critical and unacceptable. External executable should not be used in tests (git in this case)

Reply:
Fixed in #1407

Reply:
@insspb you are correct ... and way back then, I might have been lazy to use mocks ;)
Thank you  for picking up the ball and I am glad this was of some use after all.